salt/backups/templates/restore_mysql.sh

```diff
@@ -2,12 +2,13 @@
 set -e
 
 {% set backupdir = '/backups/{}'.format(settings.get('directory', 'mysql')) %}
+{% set cachedir = '/backups/.cache/{}'.format(settings.get('directory', 'mysql')) %}
 mkdir -p {{ backupdir }}
 
 PASSPHRASE={{ settings.duplicity_passphrase }} /usr/bin/duplicity restore \
           --s3-use-server-side-encryption s3+http://odl-operations-backups/{{ settings.get('directory', 'mysql') }}/ \
-          --archive-dir /backups/.cache \
-          --force --tempdir /backups  {{ backupdir }}/
+          {{ settings.database }}/ --archive-dir {{ cachedir }} \
```

Comment:
My thinking is to map each database (edxapp, xqueue) separately in case we want to backup/restore one or the other separately.